app/presenters/hyrax/version_list_presenter.rb

```diff
@@ -24,7 +24,7 @@ def self.for(file_set:)
                       else
                         Hyrax::FileSetFileService.new(file_set: file_set).original_file
                       end
-      new(original_file&.versions&.all.to_a)
+      new(original_file ? Hyrax::VersioningService.new(resource: original_file).versions : [])
```

Comment:
would be nice if the `VersioningService` handled the nil case. `Hyrax::VersioningService.new(resource: nil)).versions #=> []`, then you could squash out this conditional.

Reply:
yeah i was debating which would be better; i can go that route